pocs/utils/images/__init__.py

```diff
@@ -300,7 +300,7 @@ def _print(msg):
 
     img_path = os.path.join(dir_name, '*.fz')
```

Comment:
Maybe use `os.path.normpath()` to tidy the path (i.e. to remove `//` or `/a/../`).